src/js/themes/darkTheme/colors.ts

```diff
@@ -39,9 +39,11 @@ export const colors: ThemeColors = {
   },
   primary: {
     default: '#1098FC',
+    defaultHover: '#0092FA',
+    defaultPressed: '#54B6FC',
     alternative: '#43AEFC',
     muted: '#1098FC26',
-    inverse: '#FCFCFC',
+    inverse: '#141618',
```

Comment:
Matches figma and has css in js and css variable token

![Screenshot 2024-01-18 at 5 58 25 PM](https://github.com/MetaMask/design-tokens/assets/8112138/e8996d12-5936-4835-87ca-a64cd765a4ef)